zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -443,7 +472,25 @@ function ProjectDetails(props) {
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
         >
-          <img className={classes.enlargedImageStyle} src={enlarged_image_url} alt={`${project.title}`} />
+           
+          <IconButton
+            className={classes.cancelEnlargedImageBtn}
+            onClick={() =>
+              setState({
+                ...state,
+                open_enlarged_image_dialog: !open_enlarged_image_dialog,
+              })
+            }
+            aria-label="close"
+            >
+            <CloseIcon style={{ color: 'white' }} />
+          </IconButton>
+
+          <div className={classes.enlargedImageContainer} >
+            <NextOrPreviousButton direction={'left'} onClick={()=>handleSetState(handleGoToNextOrPreviousImage('decrement', state))} />
+            <img className={classes.enlargedImageStyle} src={project.images[state.enlarged_image_index].image_url} alt={`${project.title}`} />
+            <NextOrPreviousButton direction={'right'} onClick={()=>handleSetState(handleGoToNextOrPreviousImage('increment', state))} />
```

Comment:
I thinks its better to pass state information from setState callback to handleGoToNextOrPreviousImage to get the next Image index.